site/content/en/references/kustomize/cmd/localize/_index.md

```diff
@@ -0,0 +1,263 @@
+---
+title: "kustomize localize"
+linkTitle: "localize"
+type: docs
+weight: 9
+description: >
+  Downloads url content to a local directory, and replaces urls with paths to downloaded content
+---
+
+Disclaimer: This is an alpha command. Please see the [command proposal](https://github.com/kubernetes-sigs/kustomize/blob/master/proposals/22-04-localize-command.md)
+for full capabilities.
+
+### Description
+
+The `kustomize localize` command makes a recursive copy of a kustomization
+in which referenced urls are replaced by local paths to their downloaded content.
+The command copies files referenced under kustomization [fields](#fields).
+The copy contains files referenced both by the kustomization in question
```

Comment:
```suggestion
The copied directory contains downloaded files by the root kustomization, and all base kustomizations under it.
```

Reply:
I used `source kustomization` because I don't want readers to confuse `root kustomization` and `kustomization root`.

I also tried to avoid `downloaded` because the localized copy also contains copied local files. If you think it'd be nice to emphasize the remote files here and elaborate on copying the local files later, I can add it back in!